builder/plugins/levels-parse.js

```diff
@@ -0,0 +1,15 @@
+const tags = require('metalsmith-tags');
+
+module.exports = (path) => (files, metalsmith, done) => tags({
+  handle: 'level',
+  layout: 'tag.hbs',
+  path: `${path}/:tag.html`,
+  metadataKey: 'levels'
```

Comment:
Instead of creating levels page, can you just add levels to snippet tags?

Reply:
done